reference/7.3/Microsoft.PowerShell.Core/About/about_Parsing.md

```diff
@@ -183,11 +183,75 @@ values returned by the expressions.
 | `CMD /CECHO $AB`      | expression | 'A B' (array)   |
 | `CMD /CECHO :$AB`     | argument   | ':A B' (string) |
 
+## Handling special characters
+
+The backtick character (`` ` ``) can be used to escape any special character in
+an expression. This is most useful for escaping the argument-mode
+metacharacters that you want to use as literal characters rather than as a
+metacharacter. For example, to use the dollar sign (`$`) as a literal in an
+expandable string:
+
+```powershell
+"The value of `$ErrorActionPreference is '$ErrorActionPreference'."
+```
+
+```Output
+The value of $ErrorActionPreference is 'Continue'.
+```
+
+## Line continuation
+
+The backtick character can also be used at the end of a line to allow you to
+continue the input on the next line. This is improve the readability of a
+command that takes several parameters with long names and argument values. For
+example:
+
+```powershell
+New-AzVm `
+    -ResourceGroupName "myResourceGroupVM" `
+    -Name "myVM" `
+    -Location "EastUS" `
+    -VirtualNetworkName "myVnet" `
+    -SubnetName "mySubnet" `
+    -SecurityGroupName "myNetworkSecurityGroup" `
+    -PublicIpAddressName "myPublicIpAddress" `
+    -Credential $cred
+```
+
+However, you should avoid using line continuation.
+
+- The backtick characters can be hard see and easy to forget.
+- An extra space after the backtick breaks the line continuation. Since the
+  space is hard to see it can be difficult to find the error.
+
+PowerShell provides several ways break lines at natural points in the syntax.
+
+- After pipe characters (`|`)
```

Comment:
Nit: Not sure how to word it, but might be worth mentioning that cmdlets can be followed by a line break, whitespace, and a pipe. I'm okay with leaving it off.